### DISCUSSION TITLE
NPM repository security (not a vulnerability)

### DISCUSSION THREAD

**Discussion by znewman01** [npm/feedback#581] [Registry] [answered]
Hi! I know this is a weird place to ask this; please let me know if somewhere else is more appropriate.

I'm an academic researcher working on software repository security (along with @mnm678)---specifically, some enhancements to [TUF](https://theupdateframework.io/).

We're prefer to keep the details private until the work is a little more mature, but is there anybody appropriate for us to chat with (via any medium) about repository security, in the sense of protecting users from compromised repositories? We may have some ideas for NPM, and hearing details of your use case would help us design useful solutions.

**Comment by MylesBorins** [comment, marked as answer]
@znewman01 you might want to open an issue with support or alternatively open a ticket on hackerone if you have a POC exploit
